chain/ethereum/src/chain.rs

```diff
@@ -58,6 +58,11 @@ lazy_static! {
         .unwrap_or("100".into())
         .parse::<u64>()
         .expect("invalid GRAPH_ETHEREUM_TARGET_TRIGGERS_PER_BLOCK_RANGE");
+
+    /// Controls if firehose should be preferred over RPC if Firehose endpoints are present, if not set, the default behavior is
+    /// is kept which is to automatically favor Firehose.
+    static ref IS_FIREHOSE_PREFERRED: Option<bool> = std::env::var("GRAPH_ETHEREUM_IS_FIREHOSE_PREFERRED")
+        .map_or_else(|_| None, |input| Some(input.parse::<bool>().expect("invalid GRAPH_ETHEREUM_IS_FIREHOSE_PREFERRED")));
```

Comment:
You can use `.ok().map()` here since you're not using the first closure passed to `map_or_else`.

Reply:
Thanks, was indeed simpler with that (although it's been removed now :))